Library/QuickPulseStateManager.ts

```diff
@@ -33,12 +35,14 @@ class QuickPulseStateManager {
     private _collectors: { enable: (enable: boolean) => void }[] = [];
     private _redirectedHost: string = null;
     private _pollingIntervalHint: number = -1;
+    private _statsbeat: Statsbeat;
 
-    constructor(config: Config, context?: Context, getAuthorizationHandler?: (config: Config) => AuthorizationHandler) {
-        this.config = config;
+    constructor(client: TelemetryClient, context?: Context, getAuthorizationHandler?: (config: Config) => AuthorizationHandler) {
```

Comment:
This could be considered a breaking change, new parameters should always be added as optional in the end